webapp/src/components/Metrics/PerformanceAnalysis.tsx

```diff
@@ -184,28 +191,40 @@ const PerformanceAnalysis: React.FC<Props> = ({ jobId, pipeline }) => {
       ...NUMBER_COL_DEF,
       field: "utteranceCount",
       headerName: "Utterance Count",
-      minWidth: 160,
+      minWidth: 150,
     },
     ...ALL_OUTCOMES.map<Column<Row>>((outcome) => ({
       ...NUMBER_COL_DEF,
       field: outcome,
       headerName: OUTCOME_PRETTY_NAMES[outcome],
+      minWidth: 105,
       renderHeader: () => OutcomeIcon({ outcome }),
-      valueGetter: ({ row }) => row.outcomeCount[outcome] / row.utteranceCount,
-      valueFormatter: percentageFormatter,
+      renderCell: ({ row }: GridCellParams<undefined, Row>) =>
+        VisualBarPercentage(
+          row.outcomeCount[outcome] / row.utteranceCount,
```

Comment:
You need to keep the `valueGetter` to keep the sorting working. That should then allow you to use `value` directly in `renderCell`. Something like
```suggestion
      valueGetter: ({ row }) => row.outcomeCount[outcome] / row.utteranceCount,
      renderCell: ({ value }: GridCellParams<number>) =>
        VisualBarPercentage(
          value,
```

Reply:
yes thats right.. sorry I missed this part